blockchain/state/statedb.go

```diff
@@ -1092,3 +1103,71 @@ func (s *StateDB) GetContractStorageRoot(contractAddr common.Address) (common.Ha
 	}
 	return contract.GetStorageRoot(), nil
 }
+
+// PrepareAccessList handles the preparatory steps for executing a state transition with
+// regards to EIP-2929:
+//
+// - Add sender to access list (2929)
+// - Add destination to access list (2929)
+// - Add precompiles to access list (2929)
```

Comment:
How about adding the explanation about feepayer parameter?

Reply:
Yes, it's a good idea. i have added at https://github.com/klaytn/klaytn/pull/1705/commits/845f54426ff05505a9e2dafe37b0cd047dac3d3c.